code/game/objects/items/weapons/cards_ids.dm

```diff
@@ -135,6 +135,10 @@
 	var/blood_type = "\[UNSET\]"
 	var/dna_hash = "\[UNSET\]"
 	var/fingerprint_hash = "\[UNSET\]"
+	var/citizenship = "\[UNSET\]"
+	var/faction = "\[UNSET\]"

```

Comment:
Faction is ominous enough to where we won't use it. At least that's the present plan.

So, you can remove it from here and other places (but leave it in the prefs for the time being).